src/rpc/blockchain.cpp

```diff
@@ -1516,8 +1512,14 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
         }
     }
 
+    blockcount = std::min(blockcount, pindex->nHeight - 1);
```

Comment:
Edge case: chain has 1 block only - `blockcount` will be 0 - should be a different error?